debug-on-kubernetes.md

```diff
@@ -0,0 +1,36 @@
+# Debug support on Kubernetes cluster

```

Comment:
Could we link to this from the README please?

Reply:
Sure. But i'd like to do it in a new PR. After this wiki is checked-in, then we could reference it in README by using github cdn link `https://raw.githubusercontent.com/Azure/vscode-kubernetes-tools/master/xxx.md`.